lib/api.py

```diff
@@ -0,0 +1,53 @@
+import requests
+
+from lib import response
+from lib import errors
+
+
```

Comment:
Kill extra blank line if not needed

Reply:
This will break our CI since by flake8 standards we should have two lines
```bash
lib/api.py:5:1: E302 expected 2 blank lines, found 0
```